services/subscriptions.js

```diff
@@ -0,0 +1,96 @@
+import { WRIAPI } from 'utils/axios';
+
+/**
+ *  Get Subscriptions
+ */
+export const getSubscriptions = token =>
+  WRIAPI.get(`subscriptions?application=${process.env.APPLICATIONS}&env=${process.env.API_ENV}`, { headers: { Authorization: token } }).then(response => response.data.data);
+
+/**
+ * Creates a subscription for a pair of dataset and country
+ * @param {datasetID} ID of the dataset
+ * @param {object} Either { type; 'iso', id:'ESP' } or { type: 'geostore', id: 'sakldfa7ads0ka'}
+ * @param {string} language Two-letter locale
+ * @returns {Promise}
+ */
+export const createSubscriptionToArea = (
```

Comment:
Make it way more simple passing params ready to be sent. Services don't care about your stuff, they just trigger fetches according to your params.